site/.vitepress/theme/components/home/HeroDisplay.vue

```diff
@@ -7,7 +7,7 @@ import LinkButton from './LinkButton.vue';
         <div class="mx-auto max-w-2xl pt-20 sm:pt-28">
         <div class="hidden sm:mb-8 sm:flex sm:justify-center">
             <div class="relative rounded-full px-3 py-1 text-sm leading-6 text-gray-900 dark:text-gray-200 ring-1 ring-gray-900/10 hover:ring-gray-900/20">
-                Lightningjs 3.0 <a href="https://lightningjs.io/v3-docs/blits/getting_started/intro.html" target="_blank" class="font-semibold text-indigo-600"><span class="absolute inset-0" aria-hidden="true" />Read more <span aria-hidden="true">&rarr;</span></a>
+                Lightningjs 3.0 <a href="https://github.com/lightning-js/renderer?tab=readme-ov-file#readme" target="_blank" class="font-semibold text-indigo-600"><span class="absolute inset-0" aria-hidden="true" />Read more <span aria-hidden="true">&rarr;</span></a>
```

Comment:
Pointing folks to the renderer directly will create confusion, please leave this as it was.

Reply:
Is Lightning 3 the renderer or is it Blits? There should be a new Readme overview detailing